templates/workspace_services/azureml/terraform/providers.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.99.0"
```

Comment:
Why not 3.5?

Reply:
As we are going to come onto upgrades soon, if apply this over a preexisting bundle it would delete the AML workspace. Thought we could differentiate based on minor/major version.

Maybe I should remove the upgrade stage?

Reply:
Doh, sorry, yes, good catch. I thought that was the bundle version. I downgraded it trying to fix a bug, but will move it back to 3.5.